### ISSUE TITLE
Export issue with latest release (4.1.0)

### ISSUE THREAD

**Issue by gdetrez** [jquense/react-widgets#680]
It seems that, with the latest release, it is no longer possible to do
```js
import { DateTimePicker } from 'react-widgets'; 
```
Without using new `import/export`, this seems to be reproducible with the following (I suppose that `DateTimePicker` and the others shouldn't be `undefined`):
```sh
$ npm install react react-dom react-widgets
...
├─┬ react@16.0.0
│ ├─┬ fbjs@0.8.16
│ │ ├── core-js@1.2.7
│ │ ├─┬ isomorphic-fetch@2.2.1
│ │ │ ├─┬ node-fetch@1.7.3
│ │ │ │ ├─┬ encoding@0.1.12
│ │ │ │ │ └── iconv-lite@0.4.19
│ │ │ │ └── is-stream@1.1.0
│ │ │ └── whatwg-fetch@2.0.3
│ │ ├─┬ promise@7.3.1
│ │ │ └── asap@2.0.6
│ │ ├── setimmediate@1.0.5
│ │ └── ua-parser-js@0.7.17
│ ├─┬ loose-envify@1.3.1
│ │ └── js-tokens@3.0.2
│ ├── object-assign@4.1.1
│ └── prop-types@15.6.0
├── react-dom@16.0.0
└─┬ react-widgets@4.1.0
  ├── classnames@2.2.5
  ├── date-arithmetic@3.1.0
  ├── dom-helpers@3.2.1
  ├── invariant@2.2.2
  ├── react-component-managers@3.0.2
  ├─┬ react-prop-types@0.4.0
  │ └── warning@3.0.0
  ├─┬ react-transition-group@2.2.1
  │ ├── chain-function@1.0.0
  │ └── warning@3.0.0
  ├── spy-on-component@1.0.2
  ├── uncontrollable@4.1.0
  └── warning@2.1.0

$ node
> rw = require('react-widgets')
{ setLocalizers: [Function: setLocalizers],
  setDateLocalizer: [Function: setDate],
  setNumberLocalizer: [Function: setNumber],
  DropdownList: undefined,
  Combobox: undefined,
  Calendar: undefined,
  DatePicker: undefined,
  TimePicker: undefined,
  DateTimePicker: undefined,
  NumberPicker: undefined,
  Multiselect: undefined,
  SelectList: undefined,
  utils:
   { SlideTransitionGroup: undefined,
     SlideDownTransition: undefined } }
```

**Comment by anonmily** [comment]
Having the same issue here too! The exports are showing up as undefined.

**Comment by jquense** [comment]
Sorry folks, seems like a regression, I'll fix it fix thing tomorrow

**Comment by jquense** [comment]
fixed in 4.1.1
